src/helpers/daily-data-providers/garmin-sleep.ts

```diff
@@ -24,36 +24,34 @@ function querySleep(property: string, startDate: Date, endDate: Date, sumValues:
                 value = value / divideBy;
             }
 
-            if (!data[dataKey]) {
+            if (!data[dataKey] || data[dataKey] < value) {
```

Comment:
trying this out on my data the only way I can get two "Sleep" DDPs for a day is to edit a nights sleep. at which point I see an ENHANCED_FINAL and an AUTO_MANUAL DDP for that day.

With this logic I think the value will be whichever is less, which might be different between property types? (that is if I am drawing a graph w/ LightSleep and REMSleep the Y values might come from the different DDPs across days)



<img width="438" alt="Screenshot 2023-07-13 at 12 37 17 PM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/32501270/cc7303d3-5117-448d-85f5-5be8381c2589">

Reply:
Oh 🤦 good catch, I was trying to take the max not the min.  I actually got 2 ENHANCED_FINAL readings actually on the same night